### DISCUSSION TITLE
"Fix" difficulty expert or add a new one

### DISCUSSION THREAD

**Discussion by iamevie** [monkeytypegame/monkeytype#4538] [Ideas]
I really like the idea of a difficulty, which forces me to spell everything right but allows me to make mistakes. I think this is pretty realistic, as in real life I don't need to type every word right as long as I correct it before I send it out.

In my mind expert should exactly do that, but instead it fails, when I go to the next word.  Granted, I don't leave a word misspelled in the middle and return to it, after I'm done, so it makes sense, that it enforces me to spell the word correctly while I'm on the word but every so often, I encounter a situation, where I misspelled a character and pressed space. Thus, my run ends. If I were to press any other character, I could still delete it. This makes space a more vital mistake, which I argue shouldn't be.

**Comment by Miodec** [comment]
Stop on error: word will not allow you to go to the next word unless the current one is correct.

**Reply by iamevie** [reply]
I know, but that is not how a "real" text would work. U don't stop on an error, u have to go back.

**Reply by Miodec** [reply]
Have you tried it first? There are two modes: word and letter.
